src/Formplot/FileFormat/Vector.cs

```diff
@@ -105,6 +116,21 @@ internal static Vector Deserialize( XmlReader reader )
 			return new Vector( a.X - b.X, a.Y - b.Y, a.Z - b.Z );
 		}
 
+		public static Vector operator *( Vector a, double b )
+		{
+			return new Vector( a.X * b, a.Y * b, a.Z * b );
+		}
+
+		public static Vector operator *( double b, Vector a )
+		{
+			return new Vector( a.X * b, a.Y * b, a.Z * b );
```

Comment:
You can keep logic at one place.
```suggestion
			return a * b;
```